docs/querying/api.md

```diff
@@ -760,6 +760,76 @@ $ curl http://localhost:9090/api/v1/status/buildinfo
 
 **NOTE**: The exact returned build properties may change without notice between Prometheus versions.
 
+### TSDB Stats
+
+Thie following endpoint returns various cardinality statistics about the Prometheus TSDB:
+
+```
+GET /api/v1/status/tsdb
+```
+- **scrapeValueCountByMetricsName:**  This will provide a list of metrics names and their scaped value count.
+- **labelValueCountByLabelName:** This will provide a list of the label name and their value count.
+- **memoryUsageInBytesByLabelName** This will provide a list of the label names and memory used in bytes. Memory usage is calculated by adding the length of all values for a given label name.
+- **scrapeValueCountByLabelValuePairs** This will provide a list of label value pairs and their scaped value count.
+
+```json
+$ curl http://localhost:9090/api/v1/status/tsdb
+{
+    "status": "success",
+    "data": {
+        "queryTime": "0.331",
+        "scrapeValueCountByMetricsName": {
+            "stats": [
+                {
+                    "name": "mtail_log_lines_total",
+                    "value": 60386
+                },
+                {
+                    "name": "pilot_xds_eds_instances",
+                    "value": 72911
+                }
+            ]
+        },
+        "labelValueCountByLabelName": {
+            "stats": [
+                {
+                    "name": "task_attempt_id",
+                    "value":9607
+                },
+                {
+                    "name": "container_id",
+                    "value": 5736
+                }
+            ]
+        },
+        "memoryUsageInBytesByLabelName": {
+            "stats": [
+                {
+                    "name": "__name__",
+                    "value": 309302
+                },
+                {
+                    "name": "container_id",
+                    "value": 424933
+                }
+            ]
+        },
+        "scrapeValueCountByLabelValuePairs": {
+            "stats": [
+                {
+                    "name": "response_code=200",
+                    "value": 720701
+                },
+                {
+                    "Name": "area=data-availability",
+                    "Count": 667795
+                }
+            ]
+        }
+    }
+}
```

Comment:
This entire example is somewhat specific to our systems (such as `area=data-availability`). Would you just use the results from a locally running Prometheus scraping itself instead?